tests/test_trade.py

```diff
@@ -57,19 +57,13 @@ def test_market_init_apr_and_liquidity(self):
                         )
                         # assign the results of the init_lp agent to explicit variables
                         # market_state is used only for share_price and init_share_price
-                        # TODO: Redo this to compute the direct reserves exactly instead of calling calc_liquidity
-                        #       this ensures that the underlying function is changed and messed up at some point
-                        share_reserves_direct, bond_reserves_direct = simulator.market.pricing_model.calc_liquidity(
-                            market_state=simulator.market.market_state,
-                            target_liquidity=target_liquidity,
-                            target_apr=target_pool_apr,
-                            position_duration=simulator.market.position_duration,
-                        )
+                        # TODO: Redo this to compute the direct reserves exactly instead of calling initialize_market
+                        #       this ensures that the underlying function is not changed
                         market_direct = Market(
                             pricing_model=simulator.market.pricing_model,
                             market_state=MarketState(
-                                share_reserves=share_reserves_direct,
-                                bond_reserves=bond_reserves_direct,
+                                share_reserves=0,  # set later
+                                bond_reserves=0,  # set later
```

Comment:
can do this without passing anything in now, as per [elfpy/utils/sim_utils.py](https://github.com/element-fi/elf-simulations/pull/261/files#diff-8cae30f02fc039d47994eac409e2dd37e9ff54d6dcf4a27378488173198ceb34)